gridscale/resource_gridscale_postgresql.go

```diff
@@ -141,9 +141,9 @@ func resourceGridscalePostgreSQL() *schema.Resource {
 			},
 		},
 		Timeouts: &schema.ResourceTimeout{
-			Create: schema.DefaultTimeout(15 * time.Minute),
-			Update: schema.DefaultTimeout(15 * time.Minute),
-			Delete: schema.DefaultTimeout(15 * time.Minute),
+			Create: schema.DefaultTimeout(30 * time.Minute),
+			Update: schema.DefaultTimeout(30 * time.Minute),
+			Delete: schema.DefaultTimeout(30 * time.Minute),
```

Comment:
Fine with me in general. Esp. Kubernetes takes a while. (I've seen < 20 minutes for a 3 - 4 node cluster).

I would however make the new PostgreSQL time out waay shorter. I think normal are < 10 min here? @itakouna please correct me if I'm wrong.

Reply:
I think 15min is fine!. However, the TF will finish pulling once the resource is updated/created.

Reply:
Okay, then we do 15 min for postgresql resource and 45 min for kubernetes cluster resource. That should cover @nvthongswansea

Reply:
@bkircher done